unlock-app/src/components/content/KeychainContent.tsx

```diff
@@ -38,21 +38,19 @@ export const KeychainContent = ({
       <Head>
         <title>{pageTitle('Key Chain')}</title>
       </Head>
-      {account && (
-        <BrowserOnly>
-          <Account network={network} account={account} />
-          <KeyDetails
-            address={account.address.toLowerCase()}
-            signData={signData}
-            signatures={signatures}
-            qrEmail={qrEmail}
-          />
-        </BrowserOnly>
-      )}
-      {!account && (
-        // Default to log in form. User can toggle to signup.
-        <LogInSignUp login />
-      )}
+      <Authenticate unlockUserAccount>
+        {account && (
```

Comment:
In this case, if the children are rendered `account` is guaranteed to be there, right?

Reply:
yeah, but React complains if this is not there because the code below uses `account.address`.

I will clean that up in a future version.